Update Paolo's goals (#2801)
PostHog/posthog.com: contents/handbook/company/team/paolo-damico.md

showTitle: true
---

## Q1 2022 Goals

> - ⭐️ Build and share the **vision** for one ambitious and very **exciting** goal.

Reviewer: ```suggestion
- ⭐️ Build a clear **vision** for an ambitious goal that gets everyone **very excited**!
```

> - ⭐️ Build the strategy for one **ambiguous opportunity** to improve a company metric or achieve a company goal.

Reviewer: Not quite sure how to add this in, but this should be something that's ready for people to start working on by the end of Q1.

Author: ```suggestion
- ⭐️ Build the strategy for one **ambiguous opportunity** to improve a company metric or achieve a company goal (_should be ready to be worked on by end of Q1_)
```

showTitle: true
---

> ## Q1 2022 Goals

Reviewer: Would love to see something specific about quality in both strategy and execution

- **Product strategy.**

>   - 2x ambitious Product milestones (like Experimentation). Strategy >> Execution. Exciting/evangelism.

Reviewer: I guess this is a bit of a placeholder - but would be great if you could combine this into a single statement

Reviewer: Would love to weave in totally nailing the "why component" of every milestone with data, customer and industry context.

  - Decentralize & scale customer feedback loop with Customer Success Team.

>   - Scale (make sure it works optimally) Design work with Eng Teams.

Reviewer: Can you be more specific here? What do you mean by scale and works optimally

>   - Bring more industry context. TODO:

Reviewer: Ideally you'd have a specific objective you'd want to solve with industry context... e.g. Understand the highest quality products in our industry and identify what we can do to achieve the same level of quality... or... Evaluate the tope enterprise collaboration tools and identify the essential capabilities to make collaboration successful in PostHog...

>   - Great handle on our numbers. Discover opportunities in data (increase Discoveries from this). Excite others to do this.

Reviewer: Yep I think a great specific (but perhaps tough) goal here, would be to have an answer for why one (or more) of the company metrics have gone up or down each week during the all hands.

Reviewer 2: +1 - this _could_ be really fascinating. Maybe we lack the data volume, but we'll quickly figure that out if so.

Author: Had a chat with Marcus today and will focus on this but with a different mindset: working out a single ambiguous opportunity to improve Discoveries.

- **Product execution.**
  - Owners have all context for sprint planning, updated roadmaps and no blockers. Particularly, sprint goals should be well justified (very clear, no fluff on why) with data before sprint planning meeting.

>   - Coding and PR reviews (except UX reviews) < 20% of my time.

Reviewer: This feels like it might make more sense as a non-goal?

>   - Adjust work style to different Engineers. Less/more hands on / scoping.

Reviewer: Would be great to include in here, why you want to do this (e.g. what you expect the outcome of this to be)

>   - Increase Discoveries / active users > x%

Reviewer: Can we get a specific target for this? How will you measure it (aggregate numbers, experimentation, etc)?

Why per active user and not per active organization? If we nail this it might be at the expense of inter-organization collaboration?

## Q4 2021 Goals

### Retrospective

> Coming soon.

Reviewer: Might be worth doing a mini-face-to-face or slack survey with people to get feedback on this.

Author: Done!
